src/components/smart/OeZoneerderZonePanel.vue

```diff
@@ -0,0 +1,442 @@
+<template>
+  <div ref="zonePanelRef" :class="{ closed: !panelOpen || !props.drawPanelEnabled }" class="panel">
+    <div ref="elementRef" class="zone-panel oe-ol-control ol-control ol-unselectable">
+      <button data-cy="zonePanelControl" @click="togglePanel"><font-awesome-icon :icon="['fas', 'bars']" /></button>
+    </div>
+
+    <vl-title class="panelHeader" tag-name="h4">
+      <font-awesome-icon class="pointer" :icon="['fas', 'bars']" @click="togglePanel" />
```

Comment:
potlood icoon zal hier dudielijker zijn denk ik. eventueel extra highlight op zetten of iets groter maken om de aandacht te trekken?